BannerlordTwitch/BLTBuffet/BLTBuffet.csproj

```diff
@@ -1,158 +1,159 @@
+      <HintPath>C:\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord\Modules\SandBox\bin\Win64_Shipping_Client\SandBox.ViewModelCollection.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
+    <Reference Include="StoryMode, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>C:\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord\Modules\StoryMode\bin\Win64_Shipping_Client\StoryMode.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
+    <Reference Include="Xceed.Wpf.Toolkit, Version=4.0.0.0, Culture=neutral, PublicKeyToken=3e4669d2f30244f4">
+      <HintPath>..\packages\Extended.Wpf.Toolkit.4.0.2\lib\net40\Xceed.Wpf.Toolkit.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Actions\CharacterEffect.BLTEffectsBehaviour.cs" />
+    <Compile Include="Actions\CharacterEffect.Config.cs" />
+    <Compile Include="Actions\CharacterEffect.cs" />
+    <Compile Include="Actions\SendMessage.cs" />
+    <Compile Include="Actions\TestPfx.cs" />
+    <Compile Include="Actions\TestSfx.cs" />
+    <Compile Include="Actions\Utils\BLTBoneAttachmentsUpdateBehaviour.cs" />
+    <Compile Include="Actions\Utils\Extensions.cs" />
+    <Compile Include="BLTBuffet.cs" />
+    <Compile Include="Patches.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+    <None Include="SubModule.xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="CharacterEffectProperties.txt" />
+    <Content Include="ParticleEffects.txt" />
+    <Content Include="Sounds.txt" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="AfterClean">
+    <RemoveDir Directories="C:\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord\Modules\$(ProjectName)" />
+    <RemoveDir Directories="$(OutDir)..\..\..\..\build" />
+    <RemoveDir Directories="$(OutDir)..\..\..\..\obj" />
+  </Target>
+  <Target Name="AfterBuild">
+    <Copy SourceFiles="SubModule.xml" DestinationFolder="$(OutDir)..\.." />
+    <Copy SourceFiles="CharacterEffectProperties.txt" DestinationFolder="$(OutDir)..\.." />
+    <Copy SourceFiles="ParticleEffects.txt" DestinationFolder="$(OutDir)..\.." />
+    <Copy SourceFiles="Sounds.txt" DestinationFolder="$(OutDir)..\.." />
     <ItemGroup>
-        <Reference Include="0Harmony, Culture=neutral, PublicKeyToken=null">
-            <HintPath>C:\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord\Modules\Bannerlord.Harmony\bin\Win64_Shipping_Client\0Harmony.dll</HintPath>
-            <Private>False</Private>
-        </Reference>
-        <Reference Include="BannerlordTwitch, Culture=neutral, PublicKeyToken=null">
-          <HintPath>..\BannerlordTwitch\build\$(Configuration)\bin\Win64_Shipping_Client\BannerlordTwitch.dll</HintPath>
```

Comment:
This is wrong, it will reference the wrong dll when built in release. VS might have broken then it self if you didn't change it.